src/views/ShareCredentialSelect/ShareCredentialSelect.tsx

```diff
@@ -24,49 +26,26 @@ function MatchingIdentityCredentials({
   identity,
   onSelect,
   selected,
-  match,
   viewRef,
+  allMatching,
+  isLastIdentity,
 }: {
   identity: Identity;
   onSelect: (value: Selected) => void;
   selected?: Selected;
-  match: () => void;
   viewRef: RefObject<HTMLElement>;
-}): JSX.Element | null {
-  const data = usePopupData<ShareInput>();
-
-  const { credentialRequest } = data;
-
-  const { cTypes } = credentialRequest;
-  const cTypeHashes = cTypes.map(({ cTypeHash }) => cTypeHash);
-
-  const credentials = useIdentityCredentials(identity.did);
-
-  const matchingCredentials = credentials?.filter(
-    (credential) =>
-      cTypeHashes.includes(credential.request.claim.cTypeHash) &&
-      sameFullDid(credential.request.claim.owner, identity.did),
+  allMatching: Credential[];
```

Comment:
Would a name like `allCredentials` or `cTypeCredentials` be a better match here? (pun intended)